core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -260,6 +260,14 @@ public int getValue() {
   long getBlockSizeByte(String path) throws IOException;
 
   /**
+   * Gets the directory status.
```

Comment:
Should this also have the `don't use this method for a file`? I would prefer to exclude both since it is type safe now or include both to avoid any confusion.